docs/utils/data.rst

```diff
@@ -265,25 +263,19 @@ systems. The parallel access to the home directory can also trigger concurrency
 problems in the Astropy data cache, though we have tried to minimize these. We
 therefore recommend the following guidelines:
 
- * Do one of the following:
+ * Set ``astropy.utils.iers.conf.auto_download = False`` in your Astropy config
+   file (see :ref:`astropy_config`) or in your code so that Astropy will not
```

Comment:
[Astropy and `astropy` are not interchangeable](https://docs.astropy.org/en/latest/development/style-guide.html#capitalization)
```suggestion
 * Set ``astropy.utils.iers.conf.auto_download = False`` in your ``astropy`` config
   file (see :ref:`astropy_config`) or in your code so that ``astropy`` will not
```

Reply:
Heh, indeed, fixed.  I blame relocating old text.